scripts/orca-singularity

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+if [ "$1" = "-c" ];then
+        cmd=`echo $2 | grep "rsync\|scp"`
+        if [ -z "$cmd" ]; then
+                echo "======================================================="
+                echo "           Welcome to BCGSC ORCA Service          "
+                echo "Only direct rsync or scp file transfer commands allowed"
+                echo "======================================================="
+        else
+                exec $2
+        fi
+else
+        USER=$(id -un)
+        LOG_FILE=/orca_repo/access_log/singularity_login.log
+
+        echo "Welcome to GSC ORCA Phase 1, USER $(id -un), Your home directory is mounted under /home/$USER"
+        echo "Currently installed tools can be found via 'brew list' and 'ls ~/Software/bin' in shell"
+        echo "You can remotely transfer data via rsync/scp to your /home/$USER directory"
+#       echo "Your Hackathon project folder is mounted under /hackathon, please keep your raw data, scripts here"
+        echo " "
+
+        OVERLAY=/.orca/$USER_overlay.simg
+        ORCA_CONTAINER=orca.simg
+
+        if [ ! -f "$OVERLAY" ]; then
+                echo -e "$(date +"%D %H:%M") \t USER $(id -un) FIRST LOGIN launching new container" >> $LOG_FILE
```

Comment:
Let's always use ISO 8601 date format.
See https://xkcd.com/1179/ and https://en.wikipedia.org/wiki/ISO_8601
```sh
date --iso-8601=m
```
Use `$USER` rather than `$(id -un)`.